nbmolviz/drivers3d.py

```diff
@@ -71,30 +71,40 @@ def set_background_color(self, color, opacity=1.0):
         self.background_color = color
         self.background_opacity = opacity
 
+    def set_clipping(self, near, far):
+        self.viewer('setSlab', [float(near), float(far)])
+
     def set_color(self, color, atoms):
+        self.styles = self.get_styles_for_color(
+                color, atoms, self.styles
+        )
+
+    # Returns new styles after updating the given atoms with the given color
+    def get_styles_for_color(self, color, atoms, styles):
```

Comment:
This instance method doesn't use `self`.  Is that usually ok in Python?  Should I make it a class method, or move it to some other file outside of a class altogether?